Community roles updates (#833)
osisoft/OCS-Docs: content/set-up/roles/roles-concept.md

- Tenant Contributor – This role has read and write permissions by default.

- Tenant Data Steward – Role intended for sharing data streams within a community.*

- Tenant Viewer – Role intended for tenant guests.*

- Tenant Member – This role, which OCS assigns to all users or clients, has read access by default.

- Community Administrator – Role intended for community administration.*

__*:__ These roles are assigned no user permissions by default. They are intended to be customized for specialized user profiles within your organization. For more information, see [Tenant role profiles](#tenant-role-profiles).

For any resource, you can set access to it for specific roles rather than for specific users or clients. Manage access using Manage Permissions for the given resource. For each role, you set access to the following access types: Read, Write, Delete, and Manage Permissions. When managing permissions for the Sequential Data Store, an additional access type is available: Share. This permission allows users to share data streams from their tenant with a [community](xref:communities).

#### Tenant role profiles

> Some default tenant roles—Tenant Data Steward, Tenant Viewer, and Community Administrator—are assigned no permissions by default. Instead, these roles are intended to be customized for specialized user roles in within OSIsoft Cloud Services; you must assign these roles permissions for individual system resources to best suit your organization. 

Reviewer: ```suggestion
Some default tenant roles—Tenant Data Steward, Tenant Viewer, and Community Administrator—are assigned no permissions by default. Instead, these roles are intended to be customized for specialized user roles in within OCS; you must assign these roles permissions for individual system resources to best suit your organization. 
```

Recommended use of these roles includes:

> - Tenant Data Steward – Assign to data stewards, which are users that share data streams from the tenant's Sequential Data Store within a community.

Reviewer: ```suggestion
- Tenant Data Steward – Assign to data stewards, who share data streams from the tenant's Sequential Data Store within a community.
```

- Tenant Viewer – Assign to system guests that should have limited access.

> - Community Administrator – Assign to community administrators, which are users that moderate a community within the tenant. For more information, see [Community administrators](#community-administrators-preview).

Reviewer: ```suggestion
- Community Administrator – Assign to community administrators, who moderate a community within the tenant. For more information, see [Community administrators](#community-administrators-preview).
```

You must have the Tenant Administrator role to add and manage tenant custom roles.

### Community Member role (Preview)

The Community Member role is a role that can be shared among multiple tenants. It allows users from different tenants to access a [community](xref:communities).

When a new community is created, the founding tenant automatically adds a new Community Member role to the system, which is named using the following convention: **_\<Community Name\>_ Community Member**. For example, if you create a new community named **TEST COMMUNITY**, a new community role is added to the tenant named **TEST COMMUNITY Community Member**.

This new role is also shared with other tenants that join the community. For example, a tenant that joins **TEST COMMUNITY** will have the **TEST COMMUNITY Community Member** role added to their tenant as well. 

Users assigned the Community Member role have read permissions within the community by default. However, to allow community members to share data streams within a community, you must assign the Share permission for streams to their assigned tenant roles.

#### Community administrators (Preview)

> When a user creates or joins a new community, they are prompted to assign new community administration permissions to one or more existing roles. All users from your tenant that are assigned these roles inherit administrative permissions within the community, including the following actions:

Reviewer: When they join a community?

Author: ```suggestion
When a tenant creates or joins a new community, they are prompted to assign new community administration permissions to one or more existing roles. All users from your tenant that are assigned these roles inherit administrative permissions within the community, including the following actions:
```
